simulator/src/main/java/Environment.java

```diff
@@ -1,36 +1,94 @@
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.Map;
+import java.util.*;
```

Comment:
Does this mean we import all of java util? That may not be ideal for memory

Reply:
changed... that was intelijs doing....